optimum/neuron/utils/cache_utils.py

```diff
@@ -133,14 +133,22 @@ def create_custom_cache_repo(repo_id: str = CACHE_REPO_NAME, private: bool = Tru
 
 
 def is_private_repo(repo_id: str) -> bool:
+    """Tells whether `repo_id` is private."""
```

Comment:
Does this function check whether a repo is private to all public? If so why do we need to check if it's private to a particular user in advance?

Reply:
This function checks whether is private to the general public but public to the current user.